content/docs/1.5.4/high-availability/auto-remount-volume-by-deleting-workload-pod.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Auto Remount Volume By Deleting Workload Pod
+weight: 1
+---
+
+When the following situations occur, Longhorn automatically attempts to delete workload pods that are managed by a controller (for example, Deployment, StatefulSet, or DaemonSet). After deletion, the controller restarts the workload pod and Kubernetes handles volume reattachment and remounting.
+
+1. A volume was unexpectedly detached, possibly because of a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703), [Docker reboot](https://github.com/longhorn/longhorn/issues/686), network connectivity issue, or volume engine crash.
```

Comment:
Probably replace "Docker reboot" with "container runtime" for making it generalized?

Reply:
Fixed.